src/components/GameLeaderBoard/GameLeaderBoard.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable arrow-parens */
+import "./GameLeaderBoard.css";
+
+import { Col, Row, Space, Statistic } from "antd";
+import { useTranslation } from "react-i18next";
+
+import {
+  KeyObject,
+  RunGameData,
+  TeamLeaderBoardData,
+  TravelBingoGameData,
+} from "../../data/interfaces";
+import { calculateLeaderBoard } from "../../utils/calculate-leader-board";
+import LeaderBoard from "./LeaderBoard";
+import Podium from "./Podium";
+
+interface Props {
+  run: RunGameData & KeyObject;
+  game: TravelBingoGameData & KeyObject;
+}
+
+const GameLeaderBoard = ({ run, game }: Props) => {
+  const { t } = useTranslation();
+  const leaderBoardData = calculateLeaderBoard(run, game.challenges);
+
+  const navigate = function (team: TeamLeaderBoardData & KeyObject) {
+    window.location.href = `/${game.key}/${run.key}/${team.key}`;
+  };
+
+  return (
+    <Space direction="vertical">
+      <Row>
+        <Col span={2}></Col>
+        <Col span={10} style={{ textAlign: "center" }}>
+          <Statistic title={t("run.run-date")} value={run.date} />
+        </Col>
+        <Col span={10} style={{ textAlign: "center" }}>
+          <Statistic title={t("run.updated-at")} value={run.lastUpdate} />
```

Comment:
Would it be a good idea to parse these dates and display them in local time? We can assume that the date string is gonna be in UTC and just instantiate a date object based on that and print the local time? That would make this info more handy when the dates are few days ahead / behind the current cure. 

Also would we need to decrease the font size so that the full date would fit in the current layout? 🤔

Reply:
https://github.com/geo-quest/travel-bingo/pull/19/commits/4804dfa7213beedfd92edb2f4eb55652119fb470 ;-)